server/controllers/search/lib/entities_query_builder.js

```diff
@@ -96,15 +95,47 @@ const entitiesFields = (userLang, exact) => {
   }
   if (!exact) {
     fields.push(
-      'descriptions.*^0.25'
+      'flattenedLabels^0.25',
+      'flattenedAliases^0.25',
+      'descriptions.*^0.25',
+      'flattenedDescriptions^0.25',
+      'relationsTerms^0.25'
     )
   }
 
   return fields
 }
 
-const entitiesFlattenedFields = [
-  'flattenedLabels^0.25',
-  'flattenedAliases^0.25',
-  'flattenedDescriptions^0.1'
-]
+const getClaimFilters = claimParameter => {
+  return claimParameter
+  .split(' ')
+  .map(andCondition => {
+    const orConditions = andCondition.split('|')
```

Comment:
what about stripping spaces in case of typo, in case of messy client?
```suggestion
    const orConditions = andCondition.split(/\D\|\D/)
```
or maybe not worth the regex cost
```suggestion
    const orConditions = andCondition.split('|').map(orCondition => orCondition.trim())
```